test/BreakEventHandler.js

```diff
@@ -0,0 +1,93 @@
+var expect = require('chai').expect,
+    launcher = require('./helpers/launcher'),
+    InjectorClient = require('../lib/InjectorClient').InjectorClient,
+    BreakEventHandler = require('../lib/BreakEventHandler').BreakEventHandler,
+    ScriptManager = require('../lib/ScriptManager').ScriptManager;
+    
+describe('BreakEventHandler', function() {
+  var childProcess,
+      session;
+
+  describe('should cache events', function() {
+    it('if started in debug-brk mode', function(done) {
```

Comment:
The test names "describe should change events" and "it if started in debug-brk mode" don't read as English. Can you please rephrase? For example:

```js
describe('event cache', function() { // or 'event caching'
  it('works correctly when started in debug-brk mode', function(done) {
```

The same applies to all other test names.